packages/react/src/components/Divider/Divider.tsx

```diff
@@ -1,25 +1,19 @@
 import cl from 'clsx/lite';
-import type * as React from 'react';
 import { forwardRef } from 'react';
 
-export type DividerProps = {
-  /**
-   * The color of the divider.
-   * @default 'default'
-   */
-  color?: 'default' | 'strong' | 'subtle';
-} & React.HTMLAttributes<HTMLHRElement>;
```

Comment:
Did we ever land on this? I believe @Thunear wanted more colors, to match our neutral border colors.

Reply:
Jupp - avklart i  https://github.com/digdir/designsystemet/issues/2320#issuecomment-2324135155